Modules/ExtendedPlayerControl.cs

```diff
@@ -254,12 +254,12 @@ public static void RpcSetRoleDesync(this PlayerControl player, RoleTypes role, b
         if (player == null) return;
         if (AmongUsClient.Instance.ClientId == clientId)
         {
-            player.SetRole(role, canOverride);
```

Comment:
Leave this bool untouched please

Reply:
why, it will mess with host and modded clients